src/endpoints/getClass.ts

```diff
@@ -0,0 +1,23 @@
+import { StudentDataBase } from "./../data/Databases/StudentDataBase";
+import { CLASS } from "../data/classes/class";
+import { Request, Response } from "express";
+import { CustomError } from "../data/error/customError";
+import { ClassDataBase } from "../data/Databases/ClassDataBase";
+import { messageStatus } from "../data/error/statusCodes";
+import { v4 as generateId } from "uuid";
+import { TeacherDataBase } from "../data/Databases/TeacherDataBase";
+
+export const getClass = async (req: Request, res: Response) => {
+  try {
+    const classDB = new ClassDataBase();
+
+    const allClass = await classDB.getAllClass();
```

Comment:
O retorno dessa função é uma array, não? Nesse caso, !allClass retorna verdadeiro mesmo se a array for vazia. 
Se não houver classes e a array estiver vazia, pode-se checar isso com o .length